src/app/custom-errors.ts

```diff
@@ -68,6 +68,15 @@ export const LET_NUM_FIELD_ERRORS: ErrorMessage[] = [
   }
 ];
 
+export const PDF_FILE_ERRORS: ErrorMessage[] =
+[
+  ...CUSTOM_ERRORS,
```

Comment:
Now we are not using this code. It is deprecated. Look how it's done in core/validator or login form.